pyzeebe/worker/job_poller.py

```diff
@@ -63,3 +63,10 @@ def calculate_max_jobs_to_activate(self) -> int:
     async def stop(self):
         self.stop_event.set()
         await self.queue.join()
+
+    async def handle_max_jobs_to_activate(self):
```

Comment:
Maybe rename to `activate_max_jobs`?

Reply:
Can we move this between `poll` and `poll_once`? I think that would be a better place when reading this for the first time.

Reply:
Thanks, I've addressed both of these